packages/dhis2/test/index.test.js

```diff
@@ -142,6 +149,13 @@ describe('get', () => {
   });
 });
 
+describe('helperfunctions', () => {
+  it('should use the new tracker for enrollments', () => {
```

Comment:
More tests please!

I want a test for each of the 4 resource types, one for `tracker/<whatever>`, and at least 2 tests for values which return false (like `dataValueSet`)